### PR TITLE
Adding some text to clarify which directory user should be in before adding to local git repo for Linux Installfest step 2.4

### PR DESCRIPTION
**Pull request by sylphiae**
This is a fix for this issue that I submitted: https://github.com/railsbridge/docs/issues/501 

I looked at the change in a local deploy (screenshot attached) 
local_deploy
and also did bundle rake exec and there were no failures. I'm not very familiar with step files so please let me know if you have feedback.

### CODE REVIEW

**Comment by tjgrathwell** [comment]
Hey @sylphiae, sorry for taking so long to respond to this.

That directory path doesn't look quite right to me, your commit mentions `~/.railsbridge` when the path is more likely to be `~/railsbridge` (the `.` in front of the directory usually makes the system think it's "hidden").

Additionally, on Windows systems the student would be in `c:\sites\railsbridge\test_app`instead. Can you rephrase this in such a way that it makes sense for Windows/OSX/Linux? Maybe just ***Make sure you are in the correct directory (`test_app`)***?

**Comment by lilliealbert** [comment]
Hey @sylphiae, would you have time to make the updates that @tjgrathwell suggested above?

**Comment by tjgrathwell** [comment]
Committed an amended version of this as 8dc30786cdd12d839c1a79925fbe613440ae9e2a
